x-pack/plugins/secops/public/components/page/network/network_top_n_flow_table/index.tsx

```diff
@@ -156,6 +173,23 @@ class NetworkTopNFlowTableComponent extends React.PureComponent<NetworkTopNFlowT
     );
   }
 
+  private onChange = (criteria: Criteria) => {
+    if (criteria.sort) {
```

Comment:
Looking at the type for `criteria.sort` it looks like it is not optional and this `if statement` is not needed.

If that is true and this is not optional, I suggest removing this `if` check here. If that is not true, and `criteria.sort` can be `null/undefined` then I suggest making the type optional so we have correctly pass types that match our runtime.

If you keep this `if` statement, do you want to use `isNil` as in other parts of the code you have?

Also, does this have a unit test? It looks like a great candidate for a unit test.